### ISSUE TITLE
Configuration help - using ESLint and config from a dependency?

### ISSUE THREAD

**Issue by insin** [roadhump/SublimeLinter-eslint#105]
I've extracted my personal ESLint config and all its dependencies - including ESLint itself - [out into its own npm package](https://medium.com/@jbscript/config-tools-eslint-c85b6d48f7e2#.uc8ra92m7), so I don't have a direct local install of ESLint or a `.eslintrc` in the root of my projects any more - it's one-level removed in a dependency.

Is there a way to tell SublimeLinter-eslint to look in specific additional directories for ESLint and `.eslintrc`? In my case, it will always be in `<project root>/node_modules/eslint-config-jonnybuchanan`.

**Comment by roadhump** [comment]
Hello

You can add `--config` option with path to your file to project settings as here https://github.com/roadhump/SublimeLinter-eslint#i-want-to-use-custom-rules-global-eslintignore-file-etc

Or you can set path to external configuration inside `.eslintrc` as here http://eslint.org/docs/user-guide/configuring#extending-configuration-files

**Comment by insin** [comment]
My problem was getting the linter to find ESLint itself, rather than the config, as I prefer not to have it installed globally.

The solution I came up with was to have my configuration package install its own `.bin/eslint` command so SublimeLinter-eslint can find it, and have this dummy `eslint` find and call the real one with the appropriate `.eslintrc`.

Updated the post I linked to above with this information, too.
